extensions.json

```
@@ -1086,6 +1086,12 @@
       "checkout": "0.24.2",
       "prepublish": "npm run build-plugin"
     },
+    {
+      "id": "vscjava.vscode-spring-initializr",
+      "repository": "https://github.com/microsoft/vscode-spring-initializr",
+      "version": "0.5.0",
```

Nit: Using `"version"` without a corresponding `"checkout"` Git SHA will publish once from the default branch (even if different version).

But the default branch may contain unrelated changes (or even a different version altogether), so I think adding a `"checkout"` parameter is safer:

```suggestion
      "version": "0.5.0",
      "checkout": "v0.5.0"
```